Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs

```diff
@@ -0,0 +1,1128 @@
+                                        stepActivity.AutomationStatus = eActivityAutomationStatus.Development;//reset status because flow control param was added
+                                    }
+                                }
+                                else if (isflowControlParam == false)
+                                {
+                                    if (stepActivityVar is VariableSelectionList)
+                                    {
+                                        //change it to be string variable
+                                        stepActivity.Variables.Remove(stepActivityVar);
+                                        stepActivityVar = new VariableString();
+                                        stepActivityVar.Name = param;
+                                        ((VariableString)stepActivityVar).InitialStringValue = paramSelectedValue;
+                                        stepActivity.AddVariable(stepActivityVar);
+                                        stepActivity.AutomationStatus = eActivityAutomationStatus.Development;//reset status because flow control param was removed
+                                    }
+                                }
+                            }
+
+                            //add the variable selected value                          
+                            if (stepActivityVar is VariableSelectionList)
+                            {
+                                OptionalValue stepActivityVarOptionalVar = ((VariableSelectionList)stepActivityVar).OptionalValuesList.Where(x => x.Value == paramSelectedValue).FirstOrDefault();
+                                if (stepActivityVarOptionalVar == null)
+                                {
+                                    //no such variable value option so add it
+                                    stepActivityVarOptionalVar = new OptionalValue(paramSelectedValue);
+                                    ((VariableSelectionList)stepActivityVar).OptionalValuesList.Add(stepActivityVarOptionalVar);                                    
+                                    if (isflowControlParam == true)
+                                        stepActivity.AutomationStatus = eActivityAutomationStatus.Development;//reset status because new param value was added
+                                }
+                                //set the selected value
+                                ((VariableSelectionList)stepActivityVar).SelectedValue = stepActivityVarOptionalVar.Value;
+                            }
+                            else
+                            {
+                                //try just to set the value
+                                try
+                                {
+                                    stepActivityVar.Value = paramSelectedValue;
+                                    if (stepActivityVar is VariableString)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)